### PR TITLE
#161342831 Deploy base branch to Heroku

### PR DESCRIPTION
**Pull request by victor-nweze-deel**
#### What does this PR do?
Deploy the base branch to Heroku

#### Description of Task to be completed?
- Add a Procfile to define the entry point for Heroku
- Add an app.json file to define Heroku `review apps` configurations

#### How should this be manually tested?
- Visit the following links to view the deployed applications
> - [Staging](https://haven-ah-backend-staging.herokuapp.com/)
> - [Production](https://haven-ah-backend.herokuapp.com/)

#### What are the relevant pivotal tracker stories?
[#161342831](https://www.pivotaltracker.com/story/show/161342831)

#### Screenshots (if appropriate)
screen shot 2018-10-23 at 7 53 43 pm
screen shot 2018-10-23 at 7 54 08 pm
screen shot 2018-10-23 at 7 54 16 pm

#### Tags
@PriscillaSam @wiztemple @ucheg6 @TheoOkafor @seyi-adeleke @Chieze-Franklin

### CODE REVIEW

**Comment by PriscillaSam** [comment]
This looks okay to me.

File: app.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (seyi-adeleke) [inline app.json:1]: what is this file used for?

**Comment by seyi-adeleke** [comment]
Add reviewers using the button on the top right as opposed to tagging them in the pull request

File: app.json (hunk 1 above)
Comment (victor-nweze-deel) [reply app.json:1]: This file defines the environment variables needed to create review apps on Heroku. It is sort of the manifest for the web application. As of the moment, we don't have any scripts running.

**Comment by seyi-adeleke** [comment]
Whats the status of this PR? @ProfJigsaw

**Comment by ucheg6** [comment]
This is looking good.

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**